src/osd/PrimaryLogPG.cc

```diff
@@ -12536,7 +12536,7 @@ void PrimaryLogPG::hit_set_persist()
   map <string, bufferlist> attrs;
   attrs[OI_ATTR].claim(boi);
   attrs[SS_ATTR].claim(bss);
-  setattrs_maybe_cache(ctx->obc, ctx.get(), ctx->op_t.get(), attrs);
+  setattrs_maybe_cache(ctx->obc,  ctx->op_t.get(), attrs);
```

Comment:
Nit: Fix the unwanted space after `ctx->obc,`

Reply:
@joscollin , fixed it